tests/unit/test_command.py

```diff
@@ -54,3 +54,21 @@ def test_ls_without_all(patch_client):
 def test_ls_without_list(patch_client):
     assert usine.ls('/tmp/foo', list=False) == \
         "sh -c $'ls --all --human-readable --size /tmp/foo'"
+
+
+def test_sudo(patch_client):
+    with usine.sudo():
+        assert usine.run('whoami') == \
+            "sudo --set-home --preserve-env  sh -c $'whoami'"
+
+
+def test_sudo_user(patch_client):
+    with usine.sudo(user="me"):
+        assert usine.run('whoami') == \
+            "sudo --set-home --preserve-env --user=me --login sh -c $'whoami'"
+
+
+def test_unsudo(patch_client):
+    with usine.sudo():
+        with usine.unsudo():
+            assert usine.run('whoami') == "sh -c $'whoami'"
```

Comment:
Is `test_unsudo_user` relevant? Do we want to preserve the initial user?

Reply:
`unsudo` should totally remove the `sudo` context, and its parameters (`user` including).

Does that answer? :)

Reply:
It's suppose to be used inside a `sudo` context to temporarily run a command without this context.

Reply:
Yup, 👌